src/directives/decorators/bootstrap/select.html

```diff
@@ -9,7 +9,7 @@
           sf-changed="form"
           class="form-control {{form.fieldHtmlClass}}"
           schema-validate="form"
-          ng-options="item.value as item.name for item in form.titleMap"
+          ng-options="item.value as item.name group by item.group for item in form.titleMap"
```

Comment:
Nice tweak. I haven't had time to test it, but I guess it works even if item.group is `undefined`?